Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -1065,8 +1067,7 @@ DEBUG_CODE_END();
     // prior to starting DRAM Initialization Sequence.
     //
     MmioOr32 (PmcBase + R_PMC_PWRM_GEN_PMCON_A, B_PMC_PWRM_GEN_PMCON_A_DISB);
-
-    GpioPadConfigTable (sizeof (mGpioTablePreMemEhl) / sizeof (mGpioTablePreMemEhl[0]), mGpioTablePreMemEhl);
+    ConfigureGpio (CDATA_NO_TAG, sizeof (mGpioTablePreMemEhl) / sizeof (mGpioTablePreMemEhl[0]), (UINT8*)mGpioTablePreMemEhl);

```

Comment:
`ARRAY_SIZE (mGpioTablePreMemEhl)` instead of `sizeof (mGpioTablePreMemEhl) / sizeof (mGpioTablePreMemEhl[0])`

Reply:
Submitted patch with ARRAY_SIZE replaced